c_api/src/taichi_llvm_impl.cpp

```diff
@@ -1,8 +1,6 @@
 #include "taichi_core_impl.h"
 #include "taichi_llvm_impl.h"
 
-#ifdef TI_WITH_LLVM
```

Comment:
Woulnd't it cause any compilation problem?

Reply:
nah it's handled in cmake, when TI_WITH_LLVM is off this source file won't be added